bootstrap/kubeadm/types/upstreamv1beta4/conversion_no_fuzz_test.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package upstreamv1beta4
+
+import (
+	"testing"
+	"time"
+
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	bootstrapv1 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1beta1"
+)
+
+// This test cas been moved out of conversion_test.go because it should be run with the race detector.
+// The tests in conversion_test.go are disabled when the race detector is enabled (via "//go:build !race" above) because otherwise the fuzz tests would just time out.
```

Comment:
```suggestion
// Note: The tests in conversion_test.go are disabled when the race detector is enabled (via "//go:build !race) because otherwise the fuzz tests would just time out.
```